client/components/Layout/Images/BlurhashImage.vue

```diff
@@ -1,19 +1,19 @@
 <template>
-  <div class="absolute">
+  <div ref="imageElement">
     <div v-if="!error" ref="img" class="absolute">
-      <blurhash-canvas
-        v-if="hash"
-        key="canvas"
-        :hash="hash"
-        :width="width"
-        :height="height"
-        :punch="punch"
-        class="absolute"
-      />
+      <v-fade-transition>
+        <blurhash-canvas
+          v-if="hash"
+          :hash="hash"
+          :width="width"
+          :height="height"
+          :punch="punch"
+          class="absolute"
+        />
+      </v-fade-transition>
       <v-fade-transition>
         <img
           v-show="!loading"
-          :key="`blurhashimage-${item.Id}`"
```

Comment:
How does this work when the component is not remounted? (I.e item update or audio controls)

Reply:
It works just fine for me.